new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,679 @@
+                    + "for %s in namespace %s", domainUid, domNamespace));
+  }
+
+  private void checkPodCreated(String podName, String domainUid, String domNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podExists(podName, domainUid, domNamespace),
+            String.format("podExists failed with ApiException for %s in namespace in %s",
+                podName, domNamespace)));
+
+  }
+
+  private void checkPodReady(String podName, String domainUid, String domNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be ready in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podReady(podName, domainUid, domNamespace),
+            String.format(
+                "pod %s is not ready in namespace %s", podName, domNamespace)));
+
+  }
+
+  private void checkServiceCreated(String serviceName, String domNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for service {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                serviceName,
+                domNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> serviceExists(serviceName, null, domNamespace),
+            String.format(
+                "Service %s is not ready in namespace %s", serviceName, domainNamespace)));
+
+  }
+
+  private void checkServerReadyStatusByExec(String podName, String namespace) {
+    final V1Pod pod = assertDoesNotThrow(() -> oracle.weblogic.kubernetes.assertions.impl.Kubernetes
+        .getPod(namespace, null, podName));
+
+    if (pod != null) {
+      ExecResult execResult = assertDoesNotThrow(
+          () -> execCommand(pod, null, true, READ_STATE_COMMAND));
+      if (execResult.exitValue() == 0) {
+        logger.info("execResult: " + execResult);
+        assertEquals("RUNNING", execResult.stdout(),
+            "Expected " + podName + ", in namespace " + namespace + ", to be in RUNNING ready status");
+      } else {
+        fail("Ready command failed with exit status code: " + execResult.exitValue());
+      }
+    } else {
+      fail("Did not find pod " + podName + " in namespace " + namespace);
```

Comment:
seems to be a lot of duplicated code in this class....

Reply:
please read this https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests

Reply:
A lot of code can be moved to a common utility class.  In this class, we have only one test method rests are common utility methods.